test/eredis_parser_tests.erl

```diff
@@ -9,20 +9,16 @@
 -include("eredis.hrl").
 -include_lib("eunit/include/eunit.hrl").
 
--import(eredis_parser, [parse/2, init/0, parse_bulk/1, parse_bulk/2,
```

Comment:
I decided to remove this, since it makes for more difficult to read code.

Reply:
OK